Usagov 999 homepage updates

IsabelLaurenceau opened the pull request:
<!--- Provide a general summary of your changes in the title above -->
## Jira Task
<!--- Provide a link to the Jira ticket -->
https://cm-jira.usa.gov/browse/USAGOV-999

## Description
<!--- Describe your changes in detail -->
Updates the cards in carousel, see Jira ticket

### Testing
Please test functionality of carousel in mobile, tablet, and desktop

--Note: I discovered a bug present on prod where in mobile and tablet if using keyboard to navigate and tabbing into the carousel from the arrows it jumps to incorrect card. I created a seperate ticket for that. 

## Checklist for the Developer
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask for help! -->
- [x] A link to the JIRA ticket has been added above.
- [x] The JIRA ticket identifies the desired result of this change.
- [x] The JIRA ticket contains clear acceptance criteria.
- [x] The JIRA ticket contains clear testing steps.
- [x] The JIRA ticket contains a description of "Done".
- [x] Any preparation/installation/update steps required for this code to work properly (drush commands, scripts, configuration updates) are documented in the ticket.

## Checklist for the Peer Reviewers
- [ ] The branch name of this PR matches the project standards.
- [ ] QA steps are followed and any changes to process are updated in the ticket.
- [ ] Code Standards are followed, there are no bad practices in use.
- [ ] Config changes (if any) include only necessary changes.
- [ ] No errors in Drupal or Client Browser.
- [ ] Code has been tested locally (if possible).
- [ ] Following AC and Testing Steps verify changes work as expected.
- [ ] There are no known side-effects outside of expected behavior.

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: update card styles

still need to update sizes

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: remove photo border

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: desktop size

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: mobile sizes

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update carousel.js

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Mobile Edits

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Tablet Edits

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update _homepage.scss

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update _homepage.scss

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update _homepage.scss

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Image reset

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update _homepage.scss

IsabelLaurenceau committed:
Merge branch 'dev' into USAGOV-999-homepage-updates

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Lint fixes

akf committed:
Merge branch 'dev' into USAGOV-999-homepage-updates

akf commented:
I see a little glitch on my iPhone 12, and also in Chrome's web developer tools when viewing as an iPhone SE or iPhone 12: when viewing a card other than the first and last, the card is offset a little bit to the left, cutting off the left pixels. I added a screenshot and more notes to the Jira ticket.

akf reviewed (changes requested):
Just realized I should have added my comments as a review. See comments!

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Mobile Correct

USAGOV-999-homepage-updates: Still cutoff but going to try something and want to save progress

IsabelLaurenceau committed:
Tablet Done

USAGOV-999-homepage-updates: Tablet Done

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Desktop

IsabelLaurenceau committed:
Merge branch 'dev' into USAGOV-999-homepage-updates

IsabelLaurenceau committed:
Merge branch 'dev' into USAGOV-999-homepage-updates

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Desktop tweaks

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Update _homepage.scss

IsabelLaurenceau committed:
USAGOV-999-homepage-updates: Static Tablet Carousel

akf reviewed (approved):
Looks good now.
